### PR TITLE
[1LP][RFR] Fix view obj context

### PR DESCRIPTION
**Pull request by mshriver**
Invalid references to `self.obj`, within View classes its under `self.context['object']`

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 8192a864b92048ba13fdbf493e81556fa80e0f36:

`cfme/automate/explorer/instance.py`:
- :bangbang: Line 146:101: **E501** *line too long (105 > 100 characters)*

`cfme/automate/explorer/namespace.py`:
- :bangbang: Line 72:101: **E501** *line too long (106 > 100 characters)*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

File: cfme/automate/explorer/instance.py [hunk 1]
```diff
@@ -143,7 +143,9 @@ def all(cls, browser):
     def is_displayed(self):
         return (
             self.in_explorer and
-            self.title.text == 'Editing Automate Instance "{}"'.format(self.obj.name))
+            self.title.text == 'Editing Automate Instance "{}"'
+                               .format(self.self.context['object'].name)
```
Comment (izapolsk) [inline cfme/automate/explorer/instance.py]: typo

File: cfme/automate/explorer/klass.py [hunk 2]
```diff
@@ -97,7 +97,7 @@ class ClassEditView(ClassForm):
     def is_displayed(self):
         return (
             self.in_explorer and
-            self.title.text == 'Editing Class "{}"'.format(self.obj.name))
+            self.title.text == 'Editing Class "{}"'.format(self.self.context['object'].name))
```
Comment (izapolsk) [inline cfme/automate/explorer/klass.py]: ^^

File: cfme/dashboard.py [hunk 3]
```diff
@@ -220,7 +220,7 @@ class ParticularDashboardView(DashboardView):
     def is_displayed(self):
         return (
             super(ParticularDashboardView, self).is_displayed and
-            self.dashboards(title=self.obj.name).is_active)
+            self.dashboards(title=self.conobj.name).is_active)
```
Comment (izapolsk) [inline cfme/dashboard.py]: typo ?

Comment (mshriver) [reply cfme/dashboard.py]: fixed

**Pull request merged**